tests/unit/test_tap_kafka.py

```diff
@@ -868,6 +870,74 @@ def test_get_timestamp_from_timestamp_tuple(self):
         with pytest.raises(InvalidTimestampException):
             sync.get_timestamp_from_timestamp_tuple("not_a_tuple_or_list")
 
+    def test_initial_start_time_to_offset_reset(self):
+        """Initial start time can be one of 'latest', 'earliest' or an ISO timestamp"""
+        # Earliest should return earliest
+        assert sync.initial_start_time_to_offset_reset('earliest') == 'earliest'
```

Comment:
Can we use `self.assertEqual` for assertions, this test suite already inherits from `TestCase` which comes with these assertion methods

Reply:
all of them updated to use unittest assertions.